### PR TITLE
DSD-760: Fixing bugs for the Card component

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-760](https://jira.nypl.org/browse/DSD-760)

## This PR does the following:

- Fixes bugs found in the `Card` component.

## How has this been tested?

Locally and Storybook.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr869-zmtrf8jebr3ptligl8ml4bandeoqu3de.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/620aa75a9efcc90b3e54a53e

File: src/theme/components/card.ts [hunk 1]
```diff
@@ -128,6 +128,8 @@ const CardActions = {
 const CardContent = {
   baseStyle: {
     marginBottom: "xs",
+    width: "100%",
```
Comment (bigfishdesign13) [inline src/theme/components/card.ts]: I can’t seem to recreate a scenario in Storybook where I see these styles applied.  Where is this coming into. play?  Also, did you see the last comment from Crystal on this in the issue tracker, about not needing to implement the style (yet)?

**Review by bigfishdesign13 — COMMENTED** [review]
It looks good, but I do have some questions.  Please see the comment.

File: src/theme/components/card.ts (hunk 1 above)
Comment (EdwinGuzman) [reply src/theme/components/card.ts]: I did not. I 'll revert this and just leave the `CardHeading` update because I do need that update.

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.

**Pull request merged**